mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -627,6 +638,11 @@ class PopulateParamsXDL : public PopulateParamsBase {
   };
   const int64_t waveSize = 64;
 
+  // if driver can't use any configs in the above , use this config to do
+  // padding kernel for example , K/block is 16 , if your gemmK is  13 , we add
```

Comment:
Punctuation and capitalization of the comment is wrong. Sometimes we see gemmK and other times gemmk